src/commands/post.rs

```diff
@@ -73,9 +80,30 @@ fn run(
     let registry = registry.clone();
     let raw_args = raw_args.clone();
 
+    let content_type = call_info
+        .args
+        .get("content-type")
+        .map(|x| x.as_string().unwrap());
```

Comment:
One thing  you could do here, rather than unwrapping, is to check to see if it's the value you expect first. If it's not, you can return an error message to the user (there are examples of how to do this in other commands).

That way, if they pass in something you don't expect, Nu won't panic during the unwarp.